doc/manpages/man5/afp.conf.5.xml

```diff
@@ -2197,6 +2197,16 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term>limit size = <replaceable>BOOLEAN</replaceable> (default:
+          <emphasis>no</emphasis>) <type>(V)</type></term>
+
+          <listitem>
+            <para>Limit disk size reporting to 2GB. This can be used for older
+            Macintoshes using newer AppleShare clients.</para>
```

Comment:
What do you think about calling out this should not be used at the same time as `vol size limit`?

Users enabling both has come up a few times in the community.

Or perhaps worth considering: Can we merge the two into a single option?

Reply:
They technically don't conflict if both are used, but they can't be merged. This setting only caps the legacy 32-bit VOLPBIT_BTOTAL and VOLPBIT_BFREE values to 2GB (older Finders only handle up to 2GB and overflows with anything larger). The `vol size limit` option only limits the AFP2.2+ only 64-bit VOLPBIT_BTOTAL and VOLPBIT_XBFREE values. Its a confusing naming issue as `vol size limit` was added much later in Netatalk's life.

I did some testing with AFPBridge on the IIgs, which runs AFP2.0 (and can fake AFP2.2 if needed) over TCP/IP, and these options seem to be working properly.

Reply:
We could take this opportunity to rename `limit size` to something that makes more sense in today's context. Say `legacy vol size` or similar. Thoughts?